src/fixup/fixup.cpp

```diff
@@ -237,7 +237,7 @@ std::shared_ptr<StateDescriptor> Initialize(ParameterInput *pin) {
   params.Add("src_failure_strategy", src_failure_strategy);
 
   const bool c2p_failure_force_fixup_both =
-      pin->GetOrAddBoolean("fixup", "c2p_failure_force_fixup_both", true);
+      pin->GetOrAddBoolean("fixup", "c2p_failure_force_fixup_both", false);
```

Comment:
This option is currently broken (race condition) so by default it should be off. There is already a catch elsewhere in the code that crashes if you turn this on.